Classes/Controller/PlaceController.php

```diff
@@ -78,30 +78,21 @@ public function showAction(): ResponseInterface
 
         if (!empty($place)) {
             unset($place['photo']);
-            $place['photos'] = [];
-            $fileRepository = GeneralUtility::makeInstance(FileRepository::class);
 
             $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
                 ->getQueryBuilderForTable('sys_file_reference');
-            $fileUids = $queryBuilder
-                ->select('uid_local')
+            $place['photos'] = $queryBuilder
+                ->select('uid')
                 ->from('sys_file_reference')
                 ->where(
-                    $queryBuilder->expr()->eq('uid_foreign', $queryBuilder->createNamedParameter($place['uid'], Connection::PARAM_INT)),
+                    $queryBuilder->expr()->eq('uid_foreign', $queryBuilder->createNamedParameter($place['uid'], \PDO::PARAM_INT)),
```

Comment:
Don't use `\PDO`, you really shall use `Connection` only, see https://x.com/xperseguers/status/1765693275380371756